collectors/auth0/auth0_collector.js

```diff
@@ -27,15 +27,17 @@ const tsPaths = [
 
 const HOSTNAME_REGEXP = /^[htps]*:\/\/|\/$/gi;
 
+const STREAM_NAME = 'default';
 
 class Auth0Collector extends PawsCollector {
 
     constructor(context, creds) {
-        super(context, creds, packageJson.version);
+        super(context, creds, packageJson.version,[]);
     }
     
     pawsInitCollectionState(event, callback) {
         const initialState = {
+            stream : STREAM_NAME,
```

Comment:
not sure why we need default stream here as we already passed `[]` to the constructor.

Reply:
I will also think now it will not needed but when we had discussion ,if their is no stream set to default so that why I have added. I will remove this.